src/linter/xmlTemplate/Parser.ts

```diff
@@ -564,7 +564,9 @@ export default class Parser {
 					line: prop.start.line + 1, // Add one to align with IDEs
 					column: prop.start.column + 1,
 				};
-				if (this.#apiExtract.isAggregation(symbolName, prop.name)) {
+				if (this.#apiExtract.isProperty(symbolName, prop.name)) {
```

Comment:
This implementation skips "native" properties like "class", for example.
Everything that is not aggregation or event, should be treated as property.

Is there a reason for reverting it back?

Reply:
This PR solves false-positives for event handlers. Also checking the other attributes as property binding is a different topic, and mixing both in the same PR makes it hard to judge the differences of findings within projects that this PR creates.

I will create another PR for also checking the other attributes.

Reply:
Done via https://github.com/SAP/ui5-linter/pull/575